### DISCUSSION TITLE
Applied using university email | Rejected

### DISCUSSION THREAD

**Discussion by schezfaz** [community/community#48320] [GitHub Education]
Hi,

I applied for the github student developer pack using my Georgia Tech ID with the domain 'gatech.edu'. However, the request was denied stating that I must select my college within the university which is 'College of Computing' with the domain 'cc.gatech.edu'. However, that does not match the email domain of my student email id and corresponding github account, Please help!

![image](https://user-images.githubusercontent.com/28871211/220757093-b33fe5fd-88bd-4e7f-b758-d00508e779d1.png)

**Comment by queenofcorgis** [comment]
Hi @schezfaz,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your additional post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have removed the duplicate post, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

**Comment by samus-aran** [comment]
Hi @schezfaz 👋 

We've identified this with the internal team and would like to ask you to re-try again. Let us know how you get on 👍
